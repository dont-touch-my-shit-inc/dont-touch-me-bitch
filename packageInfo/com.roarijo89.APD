{
	"name": "App editor",
	"desc_long": "ðŸš¨APD Is a version of DLG but we changed some things updated code and better UI You can use 3 fingers and tap 3 times on the screen and the APD menu will close or openyðŸš¨<p><p><p>",
	"compatitle": "iOS 12 13 14",
	"version_web": "1.2",
	"date_web": "10/25/2020",
	"developer_name_web": "roarijo89",
	"changelog": "<strong>1.0</strong><br><br><strong>1.2</strong><br> added support to iOS 13.7 fixed some bugs<br>",
	"screenshot": null,
	"open": true,
}