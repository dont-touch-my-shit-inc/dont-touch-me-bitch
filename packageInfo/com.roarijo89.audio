{
	"name": "Audiomack",
	"desc_long": "ðŸš¨premiumðŸš¨<p><p><p>",
	"compatitle": "iOS 11 12 13 14",
	"version_web": "1.3",
	"date_web": "02/17/2021",
	"developer_name_web": "roarijo89",
	"changelog": "<strong>1.1</strong><br>premium<br><strong>1.2</strong><br>Full Premium<br>",
	"screenshot": null,
	"open": true,


}