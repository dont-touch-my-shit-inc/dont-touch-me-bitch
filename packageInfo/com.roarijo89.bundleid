{
	"name": "Bundle ID finder app version",
	"desc_long": "ðŸš¨ Press on a app to copy bundle id ðŸš¨<p><p><p>",
	"compatitle": "iOS 11 12 13 14",
	"version_web": "1.0",
	"date_web": "10/25/2020",
	"developer_name_web": "roarijo89",
	"changelog": "<strong>1.0</strong><br><br>",
	"screenshot": null,
	"open": true,

}