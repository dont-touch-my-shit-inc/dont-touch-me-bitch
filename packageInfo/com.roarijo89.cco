{
	"name": "cco",
	"desc_long": "ðŸš¨ Export your flex 3 projects without writing them you get all the code on a folderðŸš¨<p><p><p>",
	"compatitle": "iOS 11 12 13 14",
	"changelog": "<strong>1.0</strong><br><br> <strong>1.1</strong><br>updated some info<br>",
	"screenshot": null,
	"open": true,

"warning": "support <br>Flex 3 <br>Flex to theos Coco Tweaks version<br>",

}


