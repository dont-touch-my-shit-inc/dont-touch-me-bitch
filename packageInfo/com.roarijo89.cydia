{
	"name": "cydia 2021",
	"desc_long": "Cydia better UI and more<p><p><p>",
	"compatitle": "iOS 11 12 13 14",
	"version_web": "1.1",
	"date_web": "10/25/2020",
	"developer_name_web": "roarijo89",
	"changelog": "<strong>1.0</strong><br>2020<br>  <strong>1.1</strong><br>2021<br>",
	"screenshot": null,
	"open": true,
}