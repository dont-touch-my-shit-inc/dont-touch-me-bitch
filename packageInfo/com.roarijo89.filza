{
	"name": "Filza remove Activate Page popup",
	"desc_long": "ðŸš¨This will remove the activate popupðŸš¨<p><p><p>",
	"compatitle": "iOS 11 12 13 14",
	"version_web": "1.0",
	"date_web": "10/25/2020",
	"developer_name_web": "roarijo89",
	"changelog": "<strong>1.0</strong><br><br>",
	"screenshot": null,
	"open": true,
}