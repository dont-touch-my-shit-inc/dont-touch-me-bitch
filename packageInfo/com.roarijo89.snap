{
	"name": "Snapchat hack",
	"desc_long": "ðŸš¨FeaturesðŸš¨ <br> -No jailbreak detection <br> -No Screenshots Detection <br> -No Screen Record Detection <br> -No Adverts Story Ads,Lenses Ads,Â Discover Ads,Â User Stories Ads <br> -Snapchat Insights <br>-Prevent Ban / No Force Logout <p><p><p>",
	"compatitle": "iOS 11 12 13 14 15",
	"version_web": "1.5",
	"date_web": "02/18/2021",
	"developer_name_web": "roarijo89",
	"changelog": "<strong>1.0</strong><br>added no jailbreak detection<br>added no screenshots notifications<br>added prevent ban / no force logout<br> <strong>1.2</strong><br>fixed the bug that when ur signed in to the cocotweaks login popup the whole app stops working<br> <strong>1.3</strong><br>fixed some bugs<br> <strong>1.4</strong><br>added bypass by kimmo fixed some bugs<br> <strong>1.5</strong><br>Added No Screen Record Detection<br>Added Snapchat Insights<br>Added No Adverts Story Ads,Lenses Ads,Â Discover Ads,Â User Stories Ads",
	"screenshot": null,
	"open": true,

"warning": "Updated ðŸŽ‰",

}